File: kora-app-annotation-processor/src/test/java/io/koraframework/kora/app/annotation/processor/app/AppWithInheritanceComponents.java
package io.koraframework.kora.app.annotation.processor.app;

import io.koraframework.common.KoraApp;

@KoraApp
public interface AppWithInheritanceComponents extends
    AppWithInheritanceComponentsHelper.AppWithInheritanceComponentsModule1,
    AppWithInheritanceComponentsHelper.AppWithInheritanceComponentsModule2,
    AppWithInheritanceComponentsHelper.AppWithInheritanceComponentsModule3 {

}

File: core/kora-app-symbol-processor/src/test/kotlin/io/koraframework/kora/app/ksp/app/AppWithInheritanceComponents.kt
package io.koraframework.kora.app.ksp.app

import io.koraframework.common.annotation.KoraApp

@KoraApp
interface AppWithInheritanceComponents : AppWithInheritanceComponentsHelper.AppWithInheritanceComponentsModule1,
    AppWithInheritanceComponentsHelper.AppWithInheritanceComponentsModule2,
    AppWithInheritanceComponentsHelper.AppWithInheritanceComponentsModule3
